app/models/page.rb

```diff
@@ -117,13 +115,19 @@ def obsolete?
   end
 
   def redirect_page
-    if redirect_to.present?
-      @redirect_page ||= self.class.find_by_path(redirect_to, base: base, try_index: false).filename
+    lifecycle = parsed_file["lifecycle"]
+    return nil if lifecycle.nil?
+
+    if lifecycle["redirect_to"].present?
+      @redirect_page ||= self.class.find_by_path(lifecycle["redirect_to"], base: base, try_index: false).filename
```

Comment:
[suggestion] Let's add `redirect_to` to the `has_field` list above and revert this method back to the old version